app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Spot.java

```diff
@@ -0,0 +1,47 @@
+package ch.epfl.sweng.eventmanager.repository.data;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.maps.android.clustering.ClusterItem;
+
+/**
+ * Represents a place into the event
+ *
+ * @author Robin Zbinden (274236)
+ * @author Stanislas Jouven (260580)
+ */
+public class Spot implements ClusterItem {
+
+    /**
+     * The name for the spot
+     */
+    private String title;
+    /**
+     * The description of the spot
+     */
+    private SpotType spotType;
+    /**
+     * The position of the spot
+     */
+    private LatLng position;
```

Comment:
LatLng isn't a type that Firebase understands, we need to use something else (as in my EventLocation class)

Reply:
solved by d7b6b5d